src/chromecast/watch_message.cr

```diff
@@ -4,8 +4,11 @@ class Castblock::Chromecast
   struct WatchMessage
     include JSON::Serializable
 
-    getter application : Application
+    getter application : Application?
     getter media : Media?
+    getter payload : String?
+    getter payload_data : WatchMessagePayload?
+    setter payload_data : WatchMessagePayload?
```

Comment:
I'm not sure if there's a more terse way to write this, is param payload_data the same thing?

Reply:
You can use `property` which means that it is a setter and a getter. The fact that it works here while both `getter` and `setter` declare a `payload_data` is a bit strange. It seems the compiler does not bother an attribute being declared twice: https://carc.in/#/r/bfml.